src/pythermalcomfort/models.py

```diff
@@ -3724,7 +3725,7 @@ def _calculate_operative_temp_when_pmv_is_zero(
                 to = to - vpmv / 3
         return to
 
-    def _reset_setpt(self):
+    def _reset_setpt(self, par: float = 1.25):
```

Comment:
Maybe we could expand a bit the documentation here since we need to make it clear why we are calling this function.

Reply:
I added the document.